service-base/src/main/java/org/eclipse/hono/service/resourcelimits/ResourceLimitChecks.java

```diff
@@ -21,7 +22,7 @@
 /**
  * Interface to check if further connections or messages are allowed based on the configured limits.
  */
-public interface ResourceLimitChecks {
+public interface ResourceLimitChecks extends MetricsCache {
```

Comment:
Is there any need to tie these two together?
Couldn't `PrometheusBasedResourceLImitChecks` simply *also* implement `MetricsCache`?

Reply:
No, there's no need need to tie those together, especially because they do not form a "is a" relationship. I adapted the code to your proposal.